src/plugins/intel_gpu/tests/fusions/activation_fusion_test.cpp

```diff
@@ -158,6 +158,76 @@ INSTANTIATE_TEST_SUITE_P(DISABLED_fusings_gpu, activation_quantize_i8, ::testing
     activation_test_params{ CASE_ACTIVATION_3D_F32_5, 2, 3, "activation_ref" },  // FIXME - accuracy bug
 }));
 
+class prelu_quantize : public ActivationFusingTest {};
+TEST_P(prelu_quantize, basic) {
+    auto p = GetParam();
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("slope", get_mem(get_per_channel_layout(p))),
+        activation("act", "input", "slope", activation_func::relu_negative_slope),
+        data("in_low", get_mem(get_single_element_layout(p), 0)),
+        data("in_high", get_mem(get_single_element_layout(p), 1, max_random)),
+        data("out_low", get_mem(get_single_element_layout(p), -127)),
+        data("out_high", get_mem(get_single_element_layout(p), 127)),
+        quantize("quant", "act", "in_low", "in_high", "out_low", "out_high", 256, data_types::u8),
```

Comment:
```suggestion
        quantize("quant", "act", "in_low", "in_high", "out_low", "out_high", 256, data_types::i8),
```
Here and in other activation fusion tests.